src/utils/database_util.py

```diff
@@ -159,7 +159,7 @@ def table_leetcodeuser_load_by_leetcode_id(self, leetcode_id):
             return None
         return users[0]
 
-    def table_leetcodeuser_loadall(self):
+    def table_leetcodeuser_loadall(self) -> List[Document]:
```

Comment:
What type is `Document`?

Reply:
It's the class exported by TinyDB. Technically the return type of the db methods is Document rather than Dict

Reply:
Oh nice, we should eventually use that for any db method that returns one or more entries then. Better yet, the classes that we eventually make for each db table should inherit Document